src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,34 @@
+package Pages;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

Fot example

`pages`

Reply:
When I am writing Pages small(pages) it is giving me an error. 
![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/101160301/4861c346-fbe9-4942-9a91-f255cdf202c1)

Reply:
Hello

Please, re-submit homework 23 for review

Reply:
> When I am writing Pages small(pages) it is giving me an error. ![image](https://private-user-images.githubusercontent.com/101160301/298684710-4861c346-fbe9-4942-9a91-f255cdf202c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5NTAyODYsIm5iZiI6MTcwNTk0OTk4NiwicGF0aCI6Ii8xMDExNjAzMDEvMjk4Njg0NzEwLTQ4NjFjMzQ2LWZiZTktNDk0Mi05YTkxLWYyNTVjZGYyMDJjMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyMlQxODU5NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZTVmN2U3ZTFiMDBlMjYwNTMzYzBiMzkzMzk5NmQzYTc0NGYwMjA4ZWJhYzFlN2QyODZjODQ2ZTZjYjU4MzQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XpoHehTbWVj7ZtQHLHSa7ZOl7JcnG_maO8Wk0F_ahzw)

Rename refactorings https://www.jetbrains.com/help/idea/rename-refactorings.html

For example

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/9d92e665-9277-4296-bfbf-45dd886c17ae)

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/d1638c1c-cd96-4cb8-9d24-2add348ff06f)